serverless/serverless-release-notes.adoc

```diff
@@ -1,12 +1,12 @@
 include::modules/serverless-document-attributes.adoc[]
 [id="serverless-release-notes"]
-= {ServerlessProductName} Release Notes
+= {ServerlessProductName} Release notes
```

Comment:
Sorry I mislead you with that other comment, this should technically be "release notes" both lowered.